docs/_future/deploy_your_app.md

```diff
@@ -7,10 +7,249 @@ layout: tutorial
+
+If you are using Socket Mode, your `Procfile` should contain the following:
+
+```sh
+worker: node app.js
+```
+
+While those using default web connections will have the following `Procfile`:
+
+```sh
+web: node app.js
+```
+
+With this new file saved, go ahead and commit it to your Git repository:
+
+```sh
+$ git add Procfile
+$ git commit -m "Add Procfile"
+```
+
+#### Add the Git remote
+
+Code is deployed to Heroku by pushing code from your local repository to the Git remote from when [you created an app](#create-a-heroku-app).
+
+You can check if this remote was automatically added by the Heroku CLI when your app was created like so:
+
+```sh
+$ git remote -v
+# heroku  https://git.heroku.com/aqueous-escarpment-85734.git (fetch)
+# heroku  https://git.heroku.com/aqueous-escarpment-85734.git (push)
+```
+
+But if there is no `heroku` remote, you can add one as follows:
+
+```sh
+$ git remote add heroku https://git.heroku.com/aqueous-escarpment-85734.git
+```
+
+### Deploy your app {#deploy-your-app}
+
+After creating and configuring your Bolt and Heroku app, you're ready for deployment!
+
+> 💾 The [triggers created with the CLI](https://api.slack.com/future/triggers#create_cli) for the local "(dev)" version of your app will continue to fire after deploying your app!
+
+#### Push your code
+
+But before you can deploy, you should ensure that you have committed all of the source code your app needs to run!
+
+You can view your unstaged changes using `git status` and commit those changes using the following flow:
+
+```sh
+# Add all remaining unstaged changes
+$ git add .
+
+# Commit the added files
+$ git commit -m "Add in any final unstaged changes"
+```
+
+After adding and committing the source code of your app to the `main` branch of your repo, push it to your Heroku remote to start a deployment:
+
+```sh
+$ git push -u heroku main
+```
+
+Logs showing the process of your app being built by Heroku will then begin to stream in. After a moment for installing dependencies, messages indicating a successful deployment should follow as so:
+
+```sh
+remote: -----> Build succeeded!
+remote: -----> Discovering process types
+remote:        Procfile declares types     -> worker
+remote:        Default types for buildpack -> web
+
+remote: -----> Compressing...
+remote:        Done: 44.2M
+remote: -----> Launching...
+remote:        Released v6
+remote:        https://aqueous-escarpment-85734.herokuapp.com/ deployed to Heroku
+
+remote: Verifying deploy... done.
+To https://git.heroku.com/aqueous-escarpment-85734.git
+   5856f44..e322699  main -> main
+branch 'main' set up to track 'heroku/main'.
+```
+
+#### Start your app
+
+To kick things off on your Heroku app, you might have to scale up the dyno that runs your app.
```

Comment:
What are your thoughts on a quick sentence defining a dyno and then maybe linking out to an article with more info on the web / worker dynos?